faq/faq.rst

```diff
@@ -234,6 +236,33 @@ Q: How can I build LLVM compiler-rt with BTI enabled ?
        build system is up to you. The major thing to note is that the name of
        the library does not change when you enable BTI protection
 
+.. _faq_gcc_bti:
+
+Q: How can I build GCC with BTI enabled?
+========================================
+     - A GCC toolchain with BTI enabled can easily be built using Crosstool-NG:
+
+       .. code-block:: bash
+
+           $ git clone https://github.com/crosstool-ng/crosstool-ng
+           $ cd crosstool-ng
+           $ ./bootstrap && ./configure --enable-local && make
+           $ ./ct-ng aarch64-unknown-linux-gnu
+           $ cat >>.config <<_EOF_
+           CT_CC_GCC_EXTRA_CONFIG_ARRAY="--enable-standard-branch-protection"
+           CT_CC_GCC_CORE_EXTRA_CONFIG_ARRAY="--enable-standard-branch-protection"
+           _EOF_
+           $ ./ct-ng build.$(nproc)
+
+       The above commands will install the new toolchain in
+       ``~/x-tools/aarch64-unknown-linux-gnu``. You can then use this toolchain
+       to build and run OP-TEE for :ref:`qemu_v8` with full BTI support by adding
+       a few arguments to the ``make run`` command:
+
+       .. code-block:: bash
+
+           $ make CFG_CORE_BTI=y CFG_TA_BTI=y CFG_USER_TA_TARGETS=ta_arm64 AARCH64_CROSS_COMPILE=~/x-tools/aarch64-unknown-linux-gnu/bin/aarch64-linux-gnu- run
```

Comment:
A suggestion, break the line with `\` instead, so we don't get the long sideways scrollbar once rendered.

Reply:
Good idea, done.